tickets/src/components/content/EventContent.js

```diff
@@ -58,22 +58,43 @@ export class EventContent extends Component {
 
     if (!lock.address || !event.name) return null // Wait for the lock and event to load
 
-    const { name, description, location, date, links, image } = event
+    const { name, description, location, date, links = [], image } = event
```

Comment:
I assume this is because not all events have an associated links field, so this serves as the default. If that's the case, would it make more sense to insert this default where events are pulled in from storage?

Reply:
You are right! I will actually add another PR soon which adds the ability to add links.